stencil-library/src/components/business-list/business-list.tsx

```diff
@@ -48,81 +60,88 @@ export class BusinessList {
   onPageChange = (direction: string) => {
     return () => {
       this.fetchData(direction);
-    }
-  }
+    };
+  };
 
   mapProductStatusToBadge = (status: boolean) => {
-    return status && 'bg-success' || 'bg-secondary';
-  }
+    return (status && 'bg-success') || 'bg-secondary';
+  };
 
   mapBusinessType = (type: string) => {
     switch (type) {
       case 'individual':
-        return 'Individual'
+        return 'Individual';
       case 'non_profit':
-        return 'Non Profit'
+        return 'Non Profit';
       case 'for_profit':
-        return 'For Profit'
+        return 'For Profit';
       case 'government_entity':
-        return 'Government Entity'
+        return 'Government Entity';
     }
-  }
+  };
 
   mapBusinessStructure = (type: string) => {
```

Comment:
We should probably convert the swtich/cases in this file to objects at some point... not necessarily in this PR since you did not create them, but something to note I guess

Reply:
Agree with that!
@jakemer 
What are your thoughts about using just a function that will read the snake case term and convert it to normal capitalized text? 
in`utils.ts` we have `snakeCaseToHumanReadable` function that does the work.